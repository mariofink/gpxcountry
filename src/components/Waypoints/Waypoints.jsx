const Waypoints = ({ list }) => (
  <div>
    <h1>Route Builder</h1>
    <ul>
      {list.map((entry, index) => (
        <li>Waypoint {index + 1}</li>
      ))}
    </ul>
  </div>
);
export default Waypoints;
