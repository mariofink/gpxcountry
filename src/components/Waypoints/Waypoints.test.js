import React from "react";
import renderer from "react-test-renderer";
import Waypoints from "./Waypoints";

describe("Waypoints", () => {
  it("renders", () => {
    const tree = renderer
      .create(<Waypoints list={[]} onRemove={() => {}} onDrop={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
