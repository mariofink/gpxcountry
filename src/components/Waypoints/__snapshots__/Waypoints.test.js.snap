// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Waypoints renders 1`] = `
<div
  className="flex flex-col justify-between bg-gray-700 text-white p-4"
>
  <h1
    className="text-2xl font-bold border-b-4 border-gray-500 my-2 py-2"
  >
    Route Builder
  </h1>
  <ul
    className="py-8 flex-auto"
  />
</div>
`;
